client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -231,19 +231,23 @@ export const CreateEncounterModal: FC = () => {
                 return;
               }
               const startDatetime = new Date(draft?.startDatetime ?? 0);
-              if (DateUtils.addHours(startDatetime, 1).getTime() > Date.now()) {
+              const dateNow = Date.now();
+              if (
+                startDatetime.getTime() <=
+                DateUtils.addMinutes(dateNow, 5).getTime()
```

Comment:
Given can't set time, we can prob just do `if (startDatetime.getTime() > Date.now())` here?

Reply:
We can set time now >.>